frontend/src/components/studySamples/LibraryTableColumns.tsx

```diff
@@ -0,0 +1,222 @@
+import { TableColumnType } from 'antd'
```

Comment:
Unless you intend to use these definition outside they should be nested within components/studies/ unless it is an original structure. This appear to be a whole lot of definition. Keep only what is used. I'm not sure this will be useful. We'll have to check with the PM and the lab see if there is need for that level of specificity.